### PR TITLE
test: add instructions for manually testing WalletConnect

### PR DESCRIPTION
**Pull request by geigerzaehler**
Add instructions for manually testing WalletConnect.

### CODE REVIEW

**Comment by geigerzaehler** [comment]
- [ ] Add wallet network switch to the test case

File: docs/wallet-connect-testing.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+4. Reconnect with the wallet again.
+
+## Close connection modal
+
+1. Ensure you are disconnected.
+2. Click on “Connect your wallet”.
+3. Close the modal by clicking outside of it.
+4. Click on “Connect your wallet” and reconnect with a wallet again.
+
+## Switch network
```
Comment (CodeSandwich) [inline docs/wallet-connect-testing.md:42]: Do we need a similar test for switching the network in upstream settings and seeing what happens?

**Review by CodeSandwich — APPROVED** [review]
Looks good! There's no test for actually signing and sending a transaction, but I guess that other manual tests will cover that?

**Comment by geigerzaehler** [comment]
> There's no test for actually signing and sending a transaction, but I guess that other manual tests will cover that?

Yes, I was focusing on the connection for the tests. We didn’t have a lot of issues with signing and the code is pretty straight-forward.

File: docs/wallet-connect-testing.md (hunk 1 above)
Comment (geigerzaehler) [reply docs/wallet-connect-testing.md:42]: The code that handles upstream network changes does not involve wallet connect. This is something we can test automatically later.

Comment (geigerzaehler) [reply docs/wallet-connect-testing.md:42]: > are they valuable in the first place?

It’s indeed dubious whether this is valuable. We want to get rid of this anyways https://github.com/radicle-dev/radicle-upstream/issues/2002

> Maybe mocking WC could allow that thinking

Yes, that is what I’m working on.

**Comment by rudolfs** [comment]
When would we use these? Should they be part of the QA procedure?

**Comment by geigerzaehler** [comment]
> When would we use these? Should they be part of the QA procedure?

I don’t think we necessarily need to do them as part of release QA. I was running through this whenever I changed something related to wallet connect.

**Pull request merged**